files/justfiles/secureblue.just

```diff
@@ -682,3 +682,329 @@ debug-info:
+                if [[ "$resolver_subselection" == [012]* ]]; then
+                    valid_input="1"
+                else
+                    echo "That is not a valid selection."
+                fi
+            done
+            valid_input="0"
+            case "$resolver_subselection" in
+                0)
+                    resolver_ipv4_address="176.9.93.198"
+                    resolver_ipv4_address_2="176.9.1.117"
+                    resolver_ipv6_address="2a01:4f8:151:34aa::198"
+                    resolver_ipv6_address_2="2a01:4f8:141:316d::117"
+                    resolver_hostname="dnsforge.de"
+                    ;;
+                1)
+                    resolver_ipv4_address="49.12.223.2"
+                    resolver_ipv4_address_2="49.12.43.208"
+                    resolver_ipv6_address="2a01:4f8:c17:4fbc::2"
+                    resolver_ipv6_address_2="2a01:4f8:c012:ed89::208"
+                    resolver_hostname="clean.dnsforge.de"
+                    ;;
+                2)
+                    resolver_ipv4_address="49.12.222.213"
+                    resolver_ipv4_address_2="88.198.122.154"
+                    resolver_ipv6_address="2a01:4f8:c17:2c61::213"
+                    resolver_ipv6_address_2="2a01:4f8:c013:5ec0::154"
+                    resolver_hostname="hard.dnsforge.de"
+                    ;;
+            esac
+            resolver_https_address="https://$resolver_hostname/dns-query"
+            ;;
+        5)
+            echo "Setting custom resolver."
+            echo "NOTE: If the resolver does not support DoT/DoQ or DNSSEC, this process will not work."
+            echo ""
+            echo "Please provide the technical information."
+            read -p "Please enter the resolver's IP address (e.g. '1.1.1.2'): " resolver_ipv4_address
+            read -p "Does the resolver provide two distinct IP addresses (e.g. '1.1.1.2' and '1.0.0.2')? [Y/n] " resolver_has_second_ip
+            if [[ "$resolver_has_second_ip" == [Yy]* ]]; then
+                read -p "Please enter the resolver's second IP address: " resolver_ipv4_address_2
+            fi
+            read -p "Does the resolver support IPv6 (e.g. '2606:4700:4700::1112')? [Y/n] " resolver_supports_ipv6
+            if [[ "$resolver_supports_ipv6" == [Yy]* ]]; then
+                read -p "Please enter the resolver's IPv6 address: " resolver_ipv6_address
+                if [[ "$resolver_has_second_ip" == [Yy]* ]]; then
+                    read -p "Please enter the resolver's second IPv6 address: " resolver_ipv6_address_2
+                fi
+            fi
+            read -p "Please enter the second resolver's hostname (e.g. 'security.cloudflare-dns.com'): " resolver_hostname
+            ;;
+    esac
+
+    if [[ "$resolver_selection" == 0 ]]; then
+        if [[ -f "$policy_file" ]]; then
+            rm $policy_file
+            echo "Removed browser policy."
+        fi
+        cat << EOF > "$resolved_conf"
+[Resolve]
```

Comment:
on the offchance the current file changes, you could just do `cp /usr$resolved_conf $resolved_conf`